workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+Now nest two more divisions in `<div id="box" class="box">` with class as “box1” and “box2” respectively. Just place both of these in between that `<div ...>` and `</div>`.
+Now in the `<div class="box1">` tag, use 4 span tags one by one and give the class as “span1”, “span2”, “span3”, and “span4” respectively.
+
+```html
+<div class="box1">
+  <span class="span1"></span>
+  <span class="span2"></span>
+  <span class="span3"></span>
+  <span class="span4"></span>
+</div>
+```    
+
+Right now, your division should look something like this.
+
+Then inside those span tags, you can write your option and put them inside (`<b> </b>`) tags to bold your font.
+
+Now we will copy all the 4 spans and will paste it in the next division having `<div class="box2">` tag. Then just change the options of your wheel for this division.
+
+We have created all the options, now we will just add the `spin` button which we saw in the center. For that just write (`<button class="spin" onclick="myfunction()">SPIN</button>`) before the ending of your last division tag. Here we have used the onclick function that will be explained in the JavaScript part of our workshop.  
+```html
+<html>
+  <head>
+    <title>Movie Selector</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body> 
+  <div id="mainbox" class="mainbox">
+    <div id="box" class="box">
+      <div class="box1">
+        <span class="span1"><b>Iron Man</b></span>
+        <span class="span2"><b>7500</b></span>
+        <span class="span3"><b>Bat Man</b></span>
+        <span class="span4"><b>Joker</b></span>
+      </div>
+      <div class="box2">
+        <span class="span1"><b>Shoplifters</b></span>
+        <span class="span2"><b>Inception</b></span>
+        <span class="span3"><b>Deadpool</b></span>
+        <span class="span4"><b>Terminator</b></span>
+      </div>
+    </div> 
+    <button class="spin" onclick="myfunction()">SPIN</button>
+  </div>
+  <script src="script.js"></script>
+  </body>
+</html>
+```
+
+Your final code in the index.html should look something like this. Woot woot! We are all done with the HTML part of our project.
+
+## 2)	CSS
+After you have done the writing part of HTML, you'll notice that when you click the "run" button at the top, you just see a white screen with some text("options") with Spin Button. Let's now add some styles to our elements and make them look better like the way we want it to be.🤩
+ 
+Firstly, let’s go to style.css.
+
+Now add the following code:
+
+```css
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  outline: none;
+}
+
+body{
+  font-family: "Open Sans";
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
+  background: rgb(60, 60, 242);
+  background: linear-gradient(90deg, rgba(60, 60, 242, 1) 0%, rgba(98, 245, 230, 1) 52%, rgba(60, 60, 242, 1) 100%);
+  background-size: cover;
+}
+```
+      
+### Explanation: 
+3 main steps in writing CSS:
```

Comment:
This and the three bullet points don't actually explain the above code. It just gives a general explanation on how writing CSS works. It can be useful, but I don't think it works here because...what is "*"? What is "box-sizing: border-box;" and all of the other css selectors? A beginner in CSS would just copy/paste this code without understanding